src/fdc/DiskImageUtils.hh

```diff
@@ -11,11 +11,16 @@ namespace openmsx {
 
 class SectorAccessibleDisk;
 
+enum class MSXBootSectorType : uint8_t {
```

Comment:
Very much a detail. No need to change, I was just wondering...

Why did you explicitly specify the underlying type here? It's only used as a parameter (not stored in a larger struct), so the default `int` should be fine, no?

On modern CPU architectures passing/manipulating `int` is generally at least as fast and sometimes faster than smaller types. So if there's no specific reason for a smaller type (e.g. less storage) than just use `int`.  (E.g. x86 16-bit instructions have a longer encoding than 8/32 bit instructions. ARM supports more addressing modes for 32-bit load/store instructions than for 8/16-bit).

Reply:
Oh, I thought it was mandatory for some reason, I thought I got a warning about it the other day. But I must have been mistaken and there was another issue.

Reply:
Ah I probably know, I think Unreal’s reflection system requires you to be explicit about the type so I must’ve picked up the habit at work.